Makefile

```diff
@@ -110,7 +128,7 @@ all: test install lint docker-version docker-push-internal
 
 # Makefile should be edited first
 FILES_WITH_IMAGE:= .circleci/config.yml Dockerfile Dockerfile.echosrv \
-	Dockerfile.test Dockerfile.fcurl release/Dockerfile.in
+	Dockerfile.test Dockerfile.fcurl release/Dockerfile.in /etc/ssl/certs/ca-certificates.crt
```

Comment:
@danehans what is this change ? it's breaking the update tag for build image... I missed that change during review, will revert now (that list of files is all the files which reference the `fortio.build:vX` image

Reply:
I'm not 100% sure. I probably read `# Makefile should be edited first` and thought it was required to run the following `Webtest.sh` tests added by this PR:
```
docker exec $DOCKERNAME /usr/local/bin/fortio grpcping -cacert $CERT fortio.istio.io:443
docker exec $DOCKERNAME /usr/local/bin/fortio grpcping -cacert $CERT https://fortio.istio.io
```

If `Webtest.sh` passes, then it's definitely not needed. It might be a good idea to add your explanation above before `FILES_WITH_IMAGE` in the Makefile to avoid potential confusion of future contributors. I apologize for causing any problems.

Reply:
ic I'll improve the comment, it was all about changing the build image but it's not obvious